### PR TITLE
Dockerized PostgreSQL Database for Dev

### PR DESCRIPTION
**Pull request by ShawnAviles**
# Description
Created docker container for PostgreSQL database instance to be used in development. 
- Works with database seed script
- Added instructions in setup
- Updated env.example with database credentials

# Steps to test
Follow the README.md steps. Make sure you update your local `.env` variables related to the db. After following the README, the seeded db users should appear in the console when running the app (`pnpm run dev`). You should see 3 users' dummy data logged.

# Issue
Related to #50

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -6,12 +6,11 @@ generator client {
...
-    // NOTE: When using mysql or sqlserver, uncomment the @db.Text annotations in model Account below
-    // Further reading:
-    // https://next-auth.js.org/adapters/prisma#create-the-prisma-schema
-    // https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#string
-    url      = env("DATABASE_URL")
+  provider = "postgresql"
+  // Further reading:
+  // https://next-auth.js.org/adapters/prisma#create-the-prisma-schema
+  // https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#string
+  url      = env("DATABASE_URL")
```
Comment (ulrokx) [inline prisma/schema.prisma:13]: Guessing this is now formatted correctly

File: src/server/api/routers/user.ts [hunk 2]
```diff
@@ -26,4 +30,10 @@ export const userRouter = createTRPCRouter({
   getAll: protectedProcedure.query(async () => {
     return await db.user.findMany();
   }),
+
+  // TODO: Remove this example later
+  test: publicProcedure.query(async () => {
```
Comment (ulrokx) [inline src/server/api/routers/user.ts:35]: Works for me, you can probably remove these examples

**Review by ulrokx — APPROVED** [review]
Nice stuff!

File: README.md [hunk 3]
```diff
@@ -35,20 +35,40 @@
...
+     docker-compose up
+     ```
+
+   - Seed dev database
+
+     1. Generate Prisma Client
+        ```
+        pnpm run postinstall
+        ```
+     1. Generate new migration based on changes in `schema.prisma`. Note: If you'd like to discard any current schema run `npx prisma migrate reset` and then migrate
```
[the remark this answers is not in the document]
Comment (ShawnAviles) [reply README.md:58]: Good point. Ngl, I didn't understand the context and usual use cases of migrations but after seeing this comment I did more research and see how this is kinda naive. I'll update the steps accordingly to just use `db push` instead

File: prisma/schema.prisma (hunk 1 above)
Comment (ShawnAviles) [reply prisma/schema.prisma:13]: Yep, prettier

**Pull request merged**